syntax = "proto3"; 
package Room_Protobuf;
 

message RoomInfo_Response{   //210
  message User{
    string name = 1;
    int32 status = 2;
  } repeated User userinfo = 1;
}

message RoomMessage_Request{  // 113
    string content = 1;
}
  
message RoomMessage_Response{  // 213
    string content = 2;
}
  
message Start_Response{   //215
    int32 result = 1;
}